### DISCUSSION TITLE
How to import with compression? (+ set log levels)

### DISCUSSION THREAD

**Discussion by LeoniePhiline** [borgbackup/borg-import#60] [Q&A] [answered]
Hi!

I am importing from `backintime` using 

```shell
borg-import rsynchl --debug --prefix backintime /path/to/backintime/hostname/username/1/ /path/to/borg/backup/
```

I'd been wondering if there was a trick to make `borg-import` pass a `--compression` argument to `borg create`?

I found no compression environment variable in https://borgbackup.readthedocs.io/en/stable/usage/general.html#environment-variables; neither any mention of "compression" in the `borg-import` source code.

Wouldn't this be something many users could benefit from?

Side note:
I also noticed that running `borg-import --debug` does not invoke `borg create --debug`. It could be quite useful to set the log  level for the spawned borg tasks.

**Comment by ThomasWaldmann** [comment, marked as answer]
Did you try using `--create-options=...`?

Also note that borg's default compression mode is to use `lz4`, a very fast compression.

Log level: yeah, guess that is not implemented. Theoretically you could write a completely custom logging.conf and give that to borg via env var.

**Reply by LeoniePhiline** [reply]
Thank you!
